python_modules/dagster/dagster/_config/structured_config.py

```diff
@@ -15,6 +21,88 @@ class Config(BaseModel):
     """
 
 
+class cached_method:
```

Comment:
was talking about re-using cached_method in python_modules/dagster/dagster/_utils/cached_method.py not a new one

Reply:
ah, yeah, realized this, the change was pushed into the IO manager PR by mistake. updated